### PR TITLE
added translations

### PR DESCRIPTION
**Pull request by MagdaN**

### CODE REVIEW

File: src/meinberlin/meinberlin/static/i18n/meinberlin_en.json [hunk 1]
```diff
@@ -7,19 +7,30 @@
...
     "TR__MEINBERLIN_ERROR_REQUIRED_TITLE": "Please enter a proposal title",
     "TR__MEINBERLIN_I_WANT_TO_BE_PART": "I want to be a part",
-    "TR__MEINBERLIN_LOCATION": "Geo location point",
     "TR__MEINBERLIN_LOCATION_TEXT": "Address",
-    "TR__MEINBERLIN_MEIN_BERLIN_SIMPLE_PROPOSAL": "MeinBerlin Simple Proposal",
-    "TR__MEINBERLIN_MEIN_BERLIN_SIMPLE_PROPOSAL_EDIT": "Edit Proposal",
+    "TR__MEINBERLIN_MEIN_BERLIN_BPLAENE_SIMPLE_PROPOSAL": "",
+    "TR__MEINBERLIN_NAME": "Your name",
+    "TR__MEINBERLIN_STATEMENT": "Statement",
+    "TR__MEINBERLIN_STREET": "Street and Housenumber",
```
Comment (xi) [inline src/meinberlin/meinberlin/static/i18n/meinberlin_en.json]: there needs to be a space in "house number"

File: src/meinberlin/meinberlin/static/i18n/meinberlin_en.json [hunk 2]
```diff
@@ -7,19 +7,30 @@
     "TR__CREATE_PROPOSAL": "create proposal",
     "TR__CURRENT_STATE": "current state",
     "TR__END_DATE": "end date",
+    "TR__ERROR_MUST_BE_A_VALID_EMAIL_STRING": "Please enter a valid email address",
     "TR__MAP_INCOMPLETE": "Only geolocated proposals are displayed on the map. [[link:Switch to full list.]]",
     "TR__MEINBERLIN_BUDGET": "Budget",
-    "TR__MEINBERLIN_DETAIL": "Detail",
+    "TR__MEINBERLIN_CITY": "postcode and city",
+    "TR__MEINBERLIN_DETAIL": "detail",
```
Comment (xi) [inline src/meinberlin/meinberlin/static/i18n/meinberlin_en.json]: why the changed capitalization?

File: src/meinberlin/meinberlin/static/i18n/meinberlin_de.json [hunk 3]
```diff
@@ -7,23 +7,36 @@
...
     "TR__MEINBERLIN_ERROR_REQUIRED_LOCATION_TEXT": "Bitte eine Adresse angeben",
+    "TR__MEINBERLIN_ERROR_REQUIRED_NAME": "Bitte einen Namen eingeben",
+    "TR__MEINBERLIN_ERROR_REQUIRED_STATEMENT": "Bitte eine Stellungsnahme eingeben",
+    "TR__MEINBERLIN_ERROR_REQUIRED_STREET": "Bitte eine Adresse und Hausnummer angeben",
     "TR__MEINBERLIN_ERROR_REQUIRED_TITLE": "Bitte einen Titel angeben",
     "TR__MEINBERLIN_I_WANT_TO_BE_PART": "Ich möchte selber an diesem Projekt mitwirken.",
-    "TR__MEINBERLIN_LOCATION": "Sie können den Ort auf der Karte anzeigen (optional)",
     "TR__MEINBERLIN_LOCATION_TEXT": "Bitte geben Sie einen Ort für Ihr Projekt an, z.B. Grundschule XY",
-    "TR__MEINBERLIN_MEIN_BERLIN_SIMPLE_PROPOSAL": "Kiezkasse: Vorschlag einreichen",
-    "TR__MEINBERLIN_MEIN_BERLIN_SIMPLE_PROPOSAL_EDIT": "Vorschlag bearbeiten",
```
Comment (xi) [inline src/meinberlin/meinberlin/static/i18n/meinberlin_de.json]: When have these 3 been removed?

**Comment by xi** [comment]
LGTM apart from the few questins in inline comments

File: src/meinberlin/meinberlin/static/i18n/meinberlin_de.json (hunk 3 above)
Comment (MagdaN) [inline src/meinberlin/meinberlin/static/i18n/meinberlin_de.json]: I don't know.

**Comment by xi** [comment]
pushed to transifex

**Pull request merged**